pkg/api/core/run/kube_run.go

```diff
@@ -306,6 +307,38 @@ func (k *kubeRun) CreateExternalDNSKubeDeployment(opts api.CreateExternalDNSKube
 	}, nil
 }
 
+// DisableEarlyDEMUX finds and sets the aws-node's VPC CNI init container's DISABLE_TCP_EARLY_DEMUX variable to false
+func (k *kubeRun) DisableEarlyDEMUX(ctx context.Context, clusterID api.ID) error {
```

Comment:
Add link to official doc?
https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html